Index: src/model/Rey.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package model;\r\n\r\npublic class Rey{\r\n\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/model/Rey.java b/src/model/Rey.java
--- a/src/model/Rey.java	(revision c7900bcb1bb34d15d229e167080c558988187f36)
+++ b/src/model/Rey.java	(date 1772631352141)
@@ -1,6 +1,11 @@
 package model;
 
-public class Rey{
+public class Rey extends Pieza{
 
+    public Rey(int fila, int columna, Color color,int puntos){
+        super(fila,columna,color,puntos);
+    }
 
+    
+
 }
